src/projects/handmade.md

```diff
@@ -1,16 +1,28 @@
 ## Handmade Projects
 
-When you need full control and complete flexibility about your project metadata you can opt out to create a set of scripts that load the binary, setting the base address, setup the memory layout, run the analysis commands of interest, create flags, add comments, run other scripts autogenerated from SVD or other files with just one entrypoint.
+When you need full control and complete flexibility over your project metadata, you can choose to create a set of scripts that manage your binary analysis. This allows you to:
 
-The main inconvenience of these projects is that comments, flags or function names won't be saved to disk when leaving the session. We will need to manually type them into the scripts everytime we find them necessary.
+* Load the binary.
+* Set the base address.
+* Configure the memory layout.
+* Run specific analysis commands.
+* Have full control on the steps
 
-This setup requires some level of consistency from the users and also must get used to the workflow to avoid
+Note that having full control on the commands you run is important in a variety os sutations (read the section on challenges for more details).
 
 As long as handmade projects are organized in directories and files it is ideal to use it with git, allowing other people to jump into the same files and have a proper versioned.
 
-### Setup the files
+These scripts are essentially the entry points for your analysis. However, this approach comes with a trade-off: comments, flags, or function names won’t be saved automatically when exiting the session. Instead, you’ll need to manually type them into the scripts every time they are required.
 
-First of all we will create a directory to contain the binar(y/ies) you want to work on. Then you can create a **Makefile** or a shellscript with the commands to run like this:
+This workflow demands consistency and discipline from the user. It’s crucial to maintain an organized system to ensure that important metadata isn’t lost, as nothing is auto-saved unless explicitly written into the project script by the user.
```

Comment:
Why not? Can't there just be a configurable timer that auto-saves? As a user I don't want to be consistent with auto-backups, even Emacs does that :P :P